### ISSUE TITLE
Allowance management wallet 

### ISSUE THREAD

**Issue by prakhar3062** [Vikash-8090-Yadav/Solidity-Pathshala#208]
### Description

Will write the smart contract for shared wallet

**Comment by Vikash-8090-Yadav** [comment]
Can  u please describe more about this project ?

**Comment by prakhar3062** [comment]
With a wallet created by it, allowance can be received, added and withdrawn. It will also show the current balance.

**Comment by prakhar3062** [comment]
Project title
Allowance management wallet

Project Description
This project's smart contract is written in solidity for a shared wallet. This wallet can transfer funds to other users and withdraw from the wallet. It will also show the current balance. Funds can be depoisited also.

Domain
 Solidity

**Comment by Vikash-8090-Yadav** [comment]
@prakhar3062 i guess this kind of  same project is already avialable in this repo . can u please check once . !

**Comment by prakhar3062** [comment]
No, it has more functionality than that one and in the previous one, the deposit function is not complete

**Comment by abhishekbhonde** [comment]
Assign me this issue. I will solve it.

**Comment by Vikash-8090-Yadav** [comment]
@prakhar3062 ok , start working on this .

**Comment by prakhar3062** [comment]
Will be completed by today
